backend/model/tx.go

```diff
@@ -208,6 +226,45 @@ func (s Signer) String() string {
 	return fmt.Sprintf("AddrHex: %v   AddrBech32: %v \n", s.AddrHex, s.AddrBech32)
 }
 
+type MsgItem struct {
+	Type    string  `bson:"type"`
+	MsgData MsgData `bson:"msg"`
+}
+
+type MsgData struct {
+	TokenId        string `bson:"token_id"`
+	To             string `bson:"to"`
+	Family         string `bson:"family"`
+	Source         string `bson:"source"`
+	Gateway        string `bson:"gateway"`
+	Symbol         string `bson:"symbol"`
+	SymbolAtSource string `bson:"symbol_at_source"`
```

Comment:
`symbol_at_source` and `symbol_min_alias` is no longer used

Reply:
deleted